src/test/java/uk/nhs/digital/nhsconnect/lab/results/inbound/InboundMessageHandlerTest.java

```diff
@@ -66,16 +66,19 @@ void setUp() {
                 .recipient("some_recipient")
                 .sequenceNumber(SEQUENCE_NUMBER)
                 .translationTime(Instant.now())
+                .nhsAckRequested(true)
```

Comment:
I've added unit tests for this and for when exceptions are thrown.

For now I've left that mock as it is, because its being used in most of the tests in the class and would otherwise need to be duplicated in each of those methods.

Reply:
Nice, thank you!